docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -460,9 +460,6 @@ Proceed to GWAS
 
 :ref:`GloWGR GWAS functionality <gwas>` can be used to perform genome-wide association study using the phenotypic predictors to correct for polygenic effects.
 
-- **For quantitative phenotypes**, this is typically done by subtracting the predictor from the phenotype vector.
```

Comment:
We should link specifically to the offset section in the GWAS page

Reply:
I think the example notebooks are probably sufficient